ExtLibs/Maps/WPOverlay.cs

```diff
@@ -259,19 +255,30 @@ public void CreateOverlay(PointLatLngAlt home, List<Locationwp> missionitems, do
 
                     int wpno = (int) Math.Max(item.p1, 0);
                     int repeat = (int)item.p2;
+                    if (wpno > a)

```

Comment:
This needs a lengthy, borderline apologetic, downright self-deprecating explanation of why this is needed.

Reply:
Let me know if its apologetic enough or if I need to be more self deprecating